Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -953,8 +957,11 @@ final class ProductStoreTests: XCTestCase {
 
         network.simulateResponse(requestUrlSuffix: "products/\(sampleProductID)", filename: "generic_error")
         let product = sampleProduct()
-        let action = ProductAction.updateProduct(product: product) { (product, error) in
-            XCTAssertNotNil(error)
+        let action = ProductAction.updateProduct(product: product) { result in
+            guard case .failure = result else {
+                XCTFail("Unexpected result: \(result)")
+                return
+            }
             expectation.fulfill()
```

Comment:
Also here, we can use the expectation callback 👍

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/2533/commits/9f6c5b1d6666bbd75ba2fa8564cbd4bcd2bd93a7